### PR TITLE
DOC-425 part 4: FM Simple Somatic Mutation, FoundationOne Annotation and FoundationOne Variant Aggregation and Masking

### PR DESCRIPTION
**Pull request by ebusygina**
Drafts for FM Simple Somatic Mutation, FoundationOne Annotation and FoundationOne Variant Aggregation and Masking workflows. I'm less sure about these workflows as they don't have that much info.

### CODE REVIEW

File: docs/Encyclopedia/pages/FoundationOne_Variant_Aggregation_and_Masking.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# FoundationOne Variant Aggregation and Masking
+
+## Description ##
+
+FoundationOne Variant Aggregation and Masking is a somatic mutation aggregation workflow used in GDC targeted sequencing harmonization.
+
+## Overview ##
+
+FoundationOne Variant Aggregation and Masking is a pipeline used for targeted sequencing somatic mutation aggregation at the GDC. Somatic mutation aggregation is performed with FoundationOne Variant Aggregation and Masking using tumor alignments and generates simple nucleotide variation (SNV) data.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/FoundationOne_Variant_Aggregation_and_Masking.md:9]:
```suggestion
FoundationOne Variant Aggregation and Masking is a pipeline used for targeted sequencing somatic mutation aggregation at the GDC. Somatic mutation aggregation is performed by aggregating annotated somatic mutations from the FM-AD project in the GDC by case primary site.
```

File: docs/Encyclopedia/pages/FoundationOne_Annotation.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+### Input
+
+* Tumor VCF - Targeted Sequencing
+
+### Output
+
+* VCF (Data Type: Annotated Somatic Mutation)
+
+## References ##
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/FoundationOne_Annotation.md:19]:
```suggestion
```

File: docs/Encyclopedia/pages/FoundationOne_Variant_Aggregation_and_Masking.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+* Tumor VCF - Targeted Sequencing
+
+### Output
+
+* MAF (Data Type: Aggregated Somatic Mutation)
+
+## References ##
+
+1. []()
+1. []()
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/FoundationOne_Variant_Aggregation_and_Masking.md:22]:
```suggestion
```

**Review by wwysoc2 — CHANGES_REQUESTED** [review]

File: docs/Encyclopedia/pages/FM_Simple_Somatic_Mutation.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# FM Simple Somatic Mutation
+
+## Description ##
+
+FM (Foundation Medicine) Simple Somatic Mutation (SSM) workflow is a genomic profile harmonization workflow in the GDC for identifying simple somatic mutations in targeted sequencing data.
```
[the remark this answers is not in the document]
Comment (ebusygina) [reply docs/Encyclopedia/pages/FM_Simple_Somatic_Mutation.md]: changed

File: docs/Encyclopedia/pages/FM_Simple_Somatic_Mutation.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## Overview ##
+
+FM Simple Somatic Mutation is one of the pipelines used for genomic profile harmonization at the GDC. Genomic profile harmonization is performed with FM Simple Somatic Mutation using tumor alignments and generates simple nucleotide variation data.
```
[the remark this answers is not in the document]
Comment (ebusygina) [reply docs/Encyclopedia/pages/FM_Simple_Somatic_Mutation.md]: changed

File: docs/Encyclopedia/pages/FoundationOne_Annotation.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
+# FoundationOne Annotation
+
+## Description ##
+
+FoundationOne Annotation is a somatic mutation annotation workflow used in GDC targeted sequencing harmonization.
+
+## Overview ##
+
+FoundationOne Annotation is a pipeline used for targeted sequencing somatic mutation annotation at the GDC. Somatic mutation annotation is performed with FoundationOne Annotation using tumor alignments and generates simple nucleotide variation (SNV) data.
```
[the remark this answers is not in the document]
Comment (ebusygina) [reply docs/Encyclopedia/pages/FoundationOne_Annotation.md]: changed

File: docs/Encyclopedia/pages/FoundationOne_Annotation.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+* Tumor VCF - Targeted Sequencing
+
+### Output
+
+* VCF (Data Type: Annotated Somatic Mutation)
+
+## References ##
+
+1. []()
```
[the remark this answers is not in the document]
Comment (ebusygina) [reply docs/Encyclopedia/pages/FoundationOne_Annotation.md]: deleted

File: docs/Encyclopedia/pages/FoundationOne_Annotation.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+1. []()
```
[the remark this answers is not in the document]
Comment (ebusygina) [reply docs/Encyclopedia/pages/FoundationOne_Annotation.md]: deleted

File: docs/Encyclopedia/pages/FoundationOne_Variant_Aggregation_and_Masking.md (hunk 1 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/FoundationOne_Variant_Aggregation_and_Masking.md]: changed

File: docs/Encyclopedia/pages/FoundationOne_Annotation.md (hunk 2 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/FoundationOne_Annotation.md]: deleted

File: docs/Encyclopedia/pages/FoundationOne_Variant_Aggregation_and_Masking.md (hunk 3 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/FoundationOne_Variant_Aggregation_and_Masking.md]: deleted

File: docs/Encyclopedia/pages/FM_Simple_Somatic_Mutation.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+
+The FM (Foundation Medicine) Simple Somatic Mutation (SSM) workflow is a genomic profile harmonization workflow in the GDC for identifying simple somatic mutations in targeted sequencing data.
+
+## Overview ##
+
+FM Simple Somatic Mutation is the pipeline used for genomic profile harmonization for the FM-AD project at the GDC. Genomic profile harmonization is performed by lifting over simple nucleotide variation data from uploaded genomic profile data.
+
+### Input
+
+* Submitted sequencing reads
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/FM_Simple_Somatic_Mutation.md:13]:
```suggestion
* Submitted genomic profiles
```

**Review by wwysoc2 — APPROVED** [review]
There's one change I requested (I must have missed it before).  So feel free to merge after you do that.

File: docs/Encyclopedia/pages/FM_Simple_Somatic_Mutation.md (hunk 9 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/FM_Simple_Somatic_Mutation.md]: changed

**Pull request merged**